box.json

```diff
@@ -0,0 +1,29 @@
+{
+    "alias": "satis.phar",
+    "main": "bin/satis",
+    "output": "satis.phar",
+    "stub": true,
+    "directories": [ "src", "views" ],
+    "files": [
+        "LICENSE"
+    ],
+    "finder": [{
+        "name": "*.php",
```

Comment:
what about other files which may be needed (for instance the exe file necessary to ask hidden questions on windows ?)